packages/grafana-ui/src/options/builder/tooltip.tsx

```diff
@@ -42,5 +43,43 @@ export function addTooltipOptions<T extends OptionsWithTooltip>(
       settings: {
         options: sortOptions,
       },
+    })
+    .addRadio({
+      path: 'tooltip.sizing',
+      name: 'Tooltip size',
+      category,
+      settings: {
+        options: [
+          { value: TooltipSizing.Auto, label: 'Auto' },
+          { value: TooltipSizing.Manual, label: 'Manual' },
```

Comment:
I think we were thinking of just having the width / height options without the auto / manual toggle (with the defaults of the width / height being what they currently are)

Is that correct @leeoniya?

Reply:
I remember that we decided to have this option and @adela-almasan noted that in the ticket description. Also, it's a good idea to have something like a "reset" option to set the tooltip size to default values.

Reply:
yeah we dont want a toggle. we just want options for maxWidth and maxHeight.

if those are not provided we will use whatever internal defaults we choose (auto)